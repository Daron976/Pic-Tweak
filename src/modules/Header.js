import React from 'react';

const Header = () => (
  <div className="intro">
    <h1>
      Welcome to PicTweak
    </h1>
    <p>
      Filter your image to your hearts content with what we have available
    </p>
  </div>

);

export default Header;
