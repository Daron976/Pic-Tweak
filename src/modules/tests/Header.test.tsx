import renderer from 'react-test-renderer';
import Header from '../Header';

describe('Header', () => {
    test('Header page renders as expected', async () => {
      const snapHeader = renderer.create(
        <Header/>
      )
        .toJSON();
      expect(snapHeader).toMatchSnapshot();
    });
  });