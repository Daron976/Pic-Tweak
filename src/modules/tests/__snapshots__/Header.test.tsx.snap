// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Header page renders as expected 1`] = `
<div
  className="intro"
>
  <h1>
    Welcome to PicTweak
  </h1>
  <p>
    Filter your image to your hearts content with what we have available
  </p>
</div>
`;
